extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -145,14 +145,21 @@ export abstract class AzureAuth implements vscode.Disposable {
 					Logger.error(ex.originalMessageAndException);
 				}
 			} else {
+				if (ex.message) {
+					loginComplete?.reject(new AzureAuthError(`${ex.message}`, `${ex.message}`, undefined));
+					return {
+						canceled: false,
+						error: true,
+						errorCode: ex.errorCode,
+						errorMessage: ex.errorMessage || ex.message
```

Comment:
You check errorMessage here but the reject always only uses message - should you be doing the same thing there?